cmd/mpd-mpris/main.go

```diff
@@ -27,6 +29,7 @@ func init() {
 	flag.IntVar(&port, "port", 6600, "The MPD port. Only works if network is \"tcp\". If you use anything else, you should put the port inside addr yourself.")
 	flag.StringVar(&password, "pwd", "", "The MPD connection password. Leave empty for none.")
 	flag.BoolVar(&noInstance, "no-instance", false, "Set the MPDris's interface as 'org.mpris.MediaPlayer2.mpd' instead of 'org.mpris.MediaPlayer2.mpd.instance#'")
+	flag.DurationVar(&interval, "interval", time.Second/10, "How often to update the current song position. Set to 0 to never update the current song position.")
```

Comment:
Perhaps this is a saner default.
```suggestion
	flag.DurationVar(&interval, "interval", time.Second, "How often to update the current song position. Set to 0 to never update the current song position.")
```